tiledb/sm/cpp_api/query.h

```diff
@@ -333,11 +477,22 @@ class Query {
     return set_subarray(buf);
   }
 
-  /** Set the coordinate buffer for unordered queries
+  /**
+   * Set the coordinate buffer for unordered queries
```

Comment:
I would not say "unordered" queries. This is used when coordinates are either provided for writes (in both dense and sparse arrays, for either unordered or global writes), or when coordinates are retrieved for both dense/sparse array and any order in reads.